types/src/account_config/events/exchange_rate_update.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::account_config::LIBRA_MODULE_NAME;
+use anyhow::Result;
+use move_core_types::{
+    identifier::{IdentStr, Identifier},
+    move_resource::MoveResource,
+};
+use serde::{Deserialize, Serialize};
+
+/// Struct that represents a MintEvent.
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ToLBRExchangeRateUpdateEvent {
+    currency_code: Identifier,
+    new_to_lbr_exchange_rate: u64,
+}
+
+impl ToLBRExchangeRateUpdateEvent {
+    /// Exchange rates are represented as 32|32 fixed-point numbers
+    pub fn new_to_lbr_exchange_rate(&self) -> f32 {
+        (self.new_to_lbr_exchange_rate as f32) / 2f32.powf(32f32)
```

Comment:
This requires more explanation (at least for me...)

Reply:
ahhh yes.... I got this from here:
https://medium.com/incredible-coder/converting-fixed-point-to-floating-point-format-and-vice-versa-6cbc0e32544e#:~:text=Converting%20from%20fixed%2Dpoint%20to%20floating%2Dpoint&text=Divide%20the%20number%20by%202,to%20the%20power%20of%20n).

and also tested it (valid up-to rounding errors): 
https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2456fe91c9c3034233555f4aa4fe7e08